Makefile

```diff
@@ -5,12 +5,13 @@ LDFLAGS=-ldflags "-w -s -X github.com/chanzuckerberg/blessclient/pkg/util.GitSha
 
 setup:
 	curl -L https://git.io/vp6lP | BINDIR=~/.local/bin sh # gometalinter
+	curl -L https://raw.githubusercontent.com/chanzuckerberg/bff/master/download.sh | BINDIR=./_bin sh
```

Comment:
maybe I should move the `metalinter` to this dir as well in another pr

Reply:
yeah I don't have strong opinions here